tutorials/docs-00-getting-started/00_getting-started.jmd

```diff
@@ -0,0 +1,78 @@
+---
+title: Getting Started
+permalink: /:collection/:name/
+redirect_from: docs/0-gettingstarted/
+weave_options:
+  error : false
+---
+
+# Getting Started
+
+## Installation
+
+To use Turing, you need to install Julia first and then install Turing.
+
+### Install Julia
+
+You will need to install Julia 1.3 or greater, which you can get from [the official Julia website](http://julialang.org/downloads/).
+
+### Install Turing.jl
+
+Turing is an officially registered Julia package, so you can install a stable version of Turing by running the following in the Julia REPL:
+
+```julia
+using Pkg
+Pkg.add("Turing")
+```
+
+You can check if all tests pass by running (it might take a long time)
+
+```julia
```

Comment:
```suggestion
```{julia; eval=false}
```

Reply:
It might be better to disable running the entire Turing test suite while compiling this tutorial...